entry_types/scrolled/package/src/editor/models/SectionConfiguration.js

```diff
@@ -18,13 +18,20 @@ export const SectionConfiguration = Configuration.extend({
              (this.attributes.backdrop &&
               this.attributes.backdrop.image.toString().startsWith('#') ? 'color' : 'image');
```

Comment:
This line throws an exception if I try to open the section settings of the "Einsatz von Medien" section. `backdrop` is set, but, `backdrop.image` is undefined (only `backdrop.video` is present). Calling `toString` on `undefined` causes the error.

Reply:
Fixed